addons/hr/models/hr_employee.py

```diff
@@ -526,6 +526,12 @@ def _get_unusual_days(self, date_from, date_to=None):
             datetime.combine(fields.Date.from_string(date_to), time.max).replace(tzinfo=UTC)
         )
 
+    def _get_age(self, target_date=None):
```

Comment:
If this is only needed for Australian localization, why not adding it there? :thinking:

Reply:
@dasz-odoo Thanks for the PR.

Next time for minor points like this, feel free to add an independent commit on your PR (aka Australian localization), no need to create a separated PR :)

Have a nice day,